target.py

```diff
@@ -0,0 +1,117 @@
+import re
+from bs4 import BeautifulSoup
+
+
+def getTags(htmlContent):
+    soup = BeautifulSoup(htmlContent, "html.parser")
+    tags = soup.find_all()
+    tags = list(map(str, tags))
+    return tags
+
+#
+def parseTextSpacing(s):
```

Comment:
https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags

Reply:
The words don't get used anywhere. I was planning to use them but just stuck with plan text instead.